systemd/manageiq.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=ManageIQ
+After=syslog.target
+PartOf=manageiq.target
+
+[Service]
+ExecStart=/bin/sh -c "/bin/evmserver.sh start"
+ExecStop=/bin/sh -c "/bin/evmserver.sh stop"
+Type=forking
+Restart=on-failure
+KillMode=process
```

Comment:
`KillMode=process` is not recommended, `mixed` is likely what we want (SIGTERM to main process, then SIGKILL to everything in the control group (aka the workers) if it doesn't respond)